app/views/account/users/_form.html.slim

```diff
@@ -0,0 +1,8 @@
+div
+  = simple_form_for(@user, url: path) do |f|
+    = f.input :name
+    = f.input :email
+    = f.input :password
+    = f.input :password_confirmation
+    = f.button :submit
```

Comment:
for example here you can add this dropdown
but in controller parameter "role" need to be in permitted params only for admin